h/views/panels.py

```diff
@@ -31,6 +31,15 @@ def navbar(context, request):
             "?q=user:{}".format(request.authenticated_user.username))
         username = request.authenticated_user.username
 
+    if request.matched_route.name == 'activity.group_search':
+        search_url = request.route_url('activity.group_search',
+                                       pubid=request.matchdict['pubid'])
```

Comment:
This is ok, but could probably be simplified a bit with `request.current_route_url()`.